version/cmdname.go

```diff
@@ -48,3 +50,84 @@ func CmdName() string {
 	}
 	return ret
 }
+
+// findModuleInfo returns the Go module info from the executable file.
+func findModuleInfo(file string) (s string, err error) {
```

Comment:
Add a test so we detect regressions if the magic strings change?

Reply:
Done.